ci_framework/roles/edpm_build_images/molecule/default/converge.yml

```
@@ -0,0 +1,58 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Converge
+  hosts: all
+  become: true
+  roles:
+    - role: "edpm_build_images"
+  tasks:
+    - name: Ensure proper locations are created
+      block:
+        - name: Stat some files
+          register: file_stats
+          ansible.builtin.stat:
+            path: "{{ ansible_user_dir }}/ci-framework-data/{{ item }}"
+          loop:
+            - 'artifacts/edpm_images'
+        - name: Assert file status
+          assert:
+            that:
+              - item.stat.exists is defined
+              - item.stat.exists
+          loop: "{{ file_stats.results }}"
+    - name: Cleanup env
+      ansible.builtin.include_role:
```

```suggestion
      ansible.builtin.import_role:
```

We're not in a loop, we should be able to rely on import_role here. It also provides support for the `tasks_from`